### ISSUE TITLE
Frequent small releases of new software

### ISSUE THREAD

**Issue by adrianovieira** [openSUSE/osem#1995]
**I'm submitting a ..**
- [ ] Bug Report
- [X] Feature Request

**Current behavior:**
Infrequent releases of the app

**Expected correct behavior:**
Frequent small releases of fixed issues or new software functionalities

**Other infos**
Until now you've only released the version 1.0.

What if you change your release process a bit and then delivery more frequent *small* releases of OSEM?

Maybe you could release a version like e.g 2.0.0-alphaN or 2.0.0-betaN.

**Comment by palica** [comment]
+1

**Comment by Ana06** [comment]
@adrianovieira I completely agree with you. But we need to bring OSEM to an state where it makes to release it. There are many knows bugs at this point

**Comment by adrianovieira** [comment]
Thanks by your feedback.

I believe there are environments which already works with the mainline development branch (the *master* one) to support and manage their events. I used to be one of them to support my hotsite event management.
So, you probably had and still have some support of those who works with that *master branch* in production and they do some issues reporting, write code, make PR etc.

Therefore, maybe you could plan some milestones (version releases) and point which issues will integrate that milestones.

---

I recently watched a Gartner presentation and I think it fits like a glove in this situation.

So please, let me share all of you with this short talk done by Andy Kyte, Gartner R&A (~10min only). 
He delivered a presentation about "digital product management".
https://youtu.be/i8jTI-QR44w?t=2600 ~[43:15min-52:15min]

- "in a world of continuos next [..] evolve and adapts as the world changes" ~[43:50min]
- "fail early an try something else, fail early and adapt, fail early and evolve" ~[45:50min]

Kindly,
Adriano
